docs/source/data/advanced_data_catalog_usage.md

```diff
@@ -0,0 +1,258 @@
+# Advanced Data Catalog usage
+
```

Comment:
It might be helpful to have a introductory sentence here to explain what info a user can find on this page, perhaps just a list of all following sections.